GitOut/Features/Git/LocalGitRepository.cs

```diff
@@ -30,6 +30,16 @@ IProcessFactory<IGitProcess> processFactory
 
         public GitStatusResult? CachedStatus { get; private set; }
 
+        public async Task<bool> IsInsideGitFolder()
```

Comment:
Propose rename to `IsGitRepository` - I'm thinking that `GitFolder` is a bit ambiguous (it could mean the .git folder)

Reply:
The name is partly based on the command "is inside work tree", but I do agree that GitFolder is ambiguous 🤔

Reply:
just name it `IsInsideWorkTree`? 😄

Reply:
That'll work